srcalc/src/main/java/gov/va/med/srcalc/web/controller/AdminHomeController.java

```diff
@@ -37,6 +50,19 @@ public AdminHomeController(final AdminService adminService)
     @RequestMapping(method = RequestMethod.GET)
     public String defaultPage(final Model model)
     {
+        final ImmutableCollection<RiskModel>  riskModels = fAdminService.getAllRiskModels();
+
+        // Use a set since there are multiple entries in the risk models list for each model.
+        //
+        Set<EditRiskModel> editRMs = new TreeSet<EditRiskModel>(  );
+
+//        fLogger.debug( "There are {} Risk Models.", riskModels.size());
+
+        for( final RiskModel rm : riskModels ) 
+        {
+        	editRMs.add( EditRiskModel.fromRiskModel( rm ) );
```

Comment:
Why do we need to wrap the RiskModel objects in EditRiskModel objects here? The Risk Models are read-only at this point. We can just use the RiskModel objects themselves as the models for the view.